2011-02-16  Samuel Thibault  <samuel.thibault@ens-lyon.org>

	* sysdeps/mach/i386/sysdep.h: Add _MACH_I386_SYSDEP_H inclusion
	protection macro.
	* sysdeps/mach/i386/thread_state.h: Add _MACH_I386_THREAD_STATE_H
	inclusion protection macro.

--- a/sysdeps/mach/i386/sysdep.h.original	2011-02-12 02:25:25.000000000 +0000
+++ b/sysdeps/mach/i386/sysdep.h	2011-02-12 02:25:54.000000000 +0000
@@ -18,6 +18,9 @@
    Software Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
    02111-1307 USA.  */
 
+#ifndef _MACH_I386_SYSDEP_H
+#define _MACH_I386_SYSDEP_H 1
+
 /* Defines RTLD_PRIVATE_ERRNO and USE_DL_SYSINFO.  */
 #include <dl-sysdep.h>
 #include <tls.h>
@@ -78,3 +81,5 @@
 #undef ENTRY
 #undef ALIGN
 #include <sysdeps/unix/i386/sysdep.h>
+
+#endif /* mach/i386/sysdep.h */
--- a/sysdeps/mach/i386/thread_state.h.original	2011-02-12 02:26:16.000000000 +0000
+++ b/sysdeps/mach/i386/thread_state.h	2011-02-12 02:26:31.000000000 +0000
@@ -17,6 +17,9 @@
    Software Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
    02111-1307 USA.  */
 
+#ifndef _MACH_I386_THREAD_STATE_H
+#define _MACH_I386_THREAD_STATE_H 1
+
 #include <mach/machine/thread_status.h>
 
 #define MACHINE_NEW_THREAD_STATE_FLAVOR	i386_THREAD_STATE
@@ -45,3 +48,5 @@ struct machine_thread_all_state
   };
 
 #include <sysdeps/mach/thread_state.h>
+
+#endif /* mach/i386/thread_state.h */
